### ISSUE TITLE
Cat and Dog Image Classifier

### ISSUE THREAD

**Issue by DANSZITO** [DANSZITO/Rock-Paper-Scissors#2]
from tensorflow.keras.preprocessing.image import ImageDataGenerator

# Image generators for train, validation, and test sets
train_data_gen = ImageDataGenerator(rescale=1./255)
val_data_gen = ImageDataGenerator(rescale=1./255)
test_data_gen = ImageDataGenerator(rescale=1./255)

# Flow from directories (make sure to change paths as needed)
train_data = train_data_gen.flow_from_directory('cats_and_dogs/train', 
                                               target_size=(IMG_HEIGHT, IMG_WIDTH), 
                                               batch_size=BATCH_SIZE, 
                                               class_mode='binary')

val_data = val_data_gen.flow_from_directory('cats_and_dogs/validation', 
                                            target_size=(IMG_HEIGHT, IMG_WIDTH), 
                                            batch_size=BATCH_SIZE, 
                                            class_mode='binary')

test_data = test_data_gen.flow_from_directory('cats_and_dogs/test', 
                                              target_size=(IMG_HEIGHT, IMG_WIDTH), 
                                              batch_size=1, 
                                              class_mode=None, 
                                              shuffle=False)  # No shuffling for test data
